client/socket-io.go

```diff
@@ -207,13 +203,15 @@ func SetSocketIOHandlers(restClient *RestClient, BTC *btc.BTCConn, ETH *eth.ETHC
 		return nearReceivers
 	})
 
+	// on socket disconnection
 	server.On(gosocketio.OnDisconnection, func(c *gosocketio.Channel) {
 		pool.log.Infof("Disconnected %s", c.Id())
```

Comment:
`log := pool.log.WithCaller(slf.CallerShort)`
and later
`log.Debugf("receivers: %v", ... )`
and
`log.Debugf("startupReceivers: %v", ...)`